feathr_project/setup.py

```diff
@@ -65,6 +65,7 @@
         "py4j<=0.10.9.7",
         "loguru<=0.6.0",
         "pandas",
+        "numpy<=1.20.3",  # pin numpy due to pyspark's deprecated np.bool access
```

Comment:
Should this one get reverted?

Reply:
I think so, but let's see if all the test can pass, then revert this one. Otherwise one of the test might fail